README.md

```diff
@@ -598,6 +598,21 @@ var auth = new taskcluster.Auth({
 });
 ```
 
+### Configuring API Base
+If you use the builtin API Client classes documented above you can configure
+the `base` when creating an instance of the client. As illustrated below:
+
+```js
+var auth = new taskcluster.Auth({
+  credentials:  {...},
+  base:      "http://whatever.com/api"
+});
+```
+
+This will be the supported way moving forward of pointing taskcluster at
+different instances of taskcluster. This can also be set by setting a
+`TASKCLUSTER_BASE` env var before importing taskcluster-client.
```

Comment:
So in our services we won't be setting `rootUrl`?

Isn't that kind of problematic?

Reply:
Ok, will update the examples in the readme to always specify this.

Reply:
It's kiinda mandatory. Things will work just the same for the heroku cluster for the time being. To talk to any other cluster it will be mandatory, yeah.